src/main/java/com/uber/cadence/samples/spring/CadenceSamplesApplication.java

```diff
@@ -0,0 +1,32 @@
+package com.uber.cadence.samples.spring;
+
+import com.uber.cadence.client.WorkflowClient;
+import com.uber.cadence.samples.spring.models.SampleMessage;
+import com.uber.cadence.samples.spring.workflows.HelloWorldWorkflow;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Bean;
+
+@SpringBootApplication
+public class CadenceSamplesApplication {
+  public static void main(String[] args) {
+    SpringApplication.run(CadenceSamplesApplication.class, args);
+  }
+
+  // Example to start a workflow
+  @Bean
+  public CommandLineRunner startHelloWorkflow(ApplicationContext ctx) {
```

Comment:
Out of scope of this PR but in the future if we have many workflows to register for CLI, we may just put this into a seperate configuration component.